init.lua

```diff
@@ -623,6 +623,47 @@ require('lazy').setup {
     },
   },
 
+  { -- Linting
+    'mfussenegger/nvim-lint',
+    event = { 'BufReadPre', 'BufNewFile' },
+    config = function()
+      local lint = require('lint')
+      lint.linters_by_ft = {
+        markdown = { 'markdownlint' },
+      }
```

Comment:
I'm not sure what would be a good approach to get this automatically installed by Mason 🤔 anyone have any ideas?

This is how we could parse `linters_by_ft` and procure a list of tools for Mason to be installed:

```lua
      local ensure_installed = {}
      for filetype, linters in pairs(lint.linters_by_ft) do
        for index, linter in ipairs(linters) do
          table.insert(ensure_installed, linter)
        end
      end
      -- TODO: install ensure_installed with mason
```